components/sysutils/net-snmp-57/Makefile

```diff
@@ -0,0 +1,188 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2011, 2016, Oracle and/or its affiliates. All rights reserved.
+#
+# net-snmp has a "special" build where only 32-bit items are installed into
+# proto area; so prefer 32-bit to preserve existing sourcing.
+#
+
+#
+## Copyright 2016 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
+#
+
+BUILD_BITS= 32_and_64
+DOXYGEN= /usr/bin/doxygen
+include ../../../make-rules/shared-macros.mk
+
+PATH=$(VPATH):$(PATH.gnu)
+
+COMPONENT_NAME=		net-snmp
+COMPONENT_VERSION=	5.7.3
+COMPONENT_FMRI=	system/management/snmp/net-snmp
+COMPONENT_PROJECT_URL=	http://www.net-snmp.org/
+COMPONENT_SRC=    $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE= $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH= \
+    sha256:12ef89613c7707dc96d13335f153c1921efc9d61d3708ef09f3fc4a7014fb4f0
+COMPONENT_ARCHIVE_URL=	http://ftp.ntua.gr/mirror/net-snmp/net-snmp/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_CLASSIFICATION= System/Enterprise Management
+COMPONENT_SUMMARY= Net-SNMP Agent files and libraries
+
+BUILD_32_and_64 += $(BUILD_DIR_32)/.docs
```

Comment:
We don't use this variable. And I'm not sure where INSTALL_TARGET and TEST_TARGET are used.

Reply:
I removed 
- INSTALL_TARGET
- TEST_TARGET 
- BUILD_32_and_64